plugin-dev/Source/Sentry/Private/Desktop/SentryEventDesktop.cpp

```diff
@@ -56,7 +57,7 @@ ESentryLevel SentryEventDesktop::GetLevel() const
 
 bool SentryEventDesktop::IsCrash() const
 {
-	return false;
+	return IsCrashEvent;
```

Comment:
That is pretty huge! But I don't get why now? Should that have worked previously? I see it being set in the `HandleBeforeCrash` but it seems unrelated to the GC.

Reply:
The fact that `IsCrash()` method for desktop events was simply a stub caught my eye only now while reworking `onCrash` handler. We don't really use its Win/Linux implementation within the SDK so I assume that's the reason why it remained unnoticed for so long.

Would you prefer to fix this in a separate PR or we're okay with merging it with other changes here?